client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+const _ = require("lodash");
```

Comment:
Well if I want to require lodash.escaperegexp I need to install it and add it to packages.json as a dependency. Still okay? Or do you prefer copy pasting?